parm/stage/analysis.yaml.j2

```diff
@@ -15,5 +15,12 @@ analysis:
         - ["{{ ICSDIR }}/{{ COMOUT_ATMOS_ANALYSIS_MEM | relpath(ROTDIR) }}/{{ RUN }}.t{{ current_cycle_HH }}z.{{ ftype }}", "{{ COMOUT_ATMOS_ANALYSIS_MEM }}"]
         {% endif %}
         {% endfor %}
+        {% if DO_JEDIATMVAR %}
+        {% for ftype in ["satbias.nc", "satbias_cov.nc", "tlapse.txt"] %}
+        {% for file in glob(ICSDIR ~ "/" ~ COMOUT_ATMOS_ANALYSIS_MEM | relpath(ROTDIR) ~ "/" ~ RUN ~ ".t" ~ current_cycle_HH ~ "z.atms_*." ~ ftype) %}
```

Comment:
A bit leery of the glob. My understanding is we'd like to avoid them as much as possible, marking files as optional if necessary.

CC @aerorahul

Reply:
We found `glob` being used in `parm/archive/gfsa.yaml`.  Thus, we followed a similar approach here.  This doesn't mean it's the best approach.  

GSI radiance bias correction files contain all _satellite_sensors_ in a single file.  JEDI differs.  JEDI currently has three files for each _satellite_sensor_.  Instead of hardwiring a list of _satellite_sensors_ to loop over, we opted to use `glob` to grab whatever JEDI radiance bias correction files are present.  This seemed more efficient and more flexible (less hardwired). 

I understand being leery of `glob`.  As @CoryMartin-NOAA , another approach is to store all JEDI radiance bias correction files in a tarball.  The tarball is what gets copied, moved, and archived.  This is doable.  It requires changes to g-w yamls, templates, and scripts.  I'm willing to do the work but would prefer to do so as a separate issue and PR instead of bundling it into this lgetkf observer and solver PR.

What do you think?

Reply:
Making a tarball just to avoid a glob seems like an unnecessary step, and just moves the issue elsewhere instead of addressing it (you still have to determine what goes in the tarball). It's more an issue or error checking:
- What bias corrections are expected?
- What should happen if one is missing? What about all of them?
- What if there is a new one? Is there a reason to process it if the corresponding obs platform isn't supported yet?

Glob can make sense in some situations, especially with homogenous data. For instance, considering a slightly different datatype, if there were a bunch of surface obs or radiosondes, where they are all going to be treated the same way and you expect the availability to vary day-to-day, those would make sense to just glob. I don't know that either of these conditions hold here.

These issues don't necessarily need to be solved here and now, but it tends to be easier to catch and design for these things up front than remember to go back and address them later.

Reply:
Is there a desire to tar/combine all sensor `satbias.nc`, `satbias_cov.nc`, and `tlapse.txt` files such that at staging one can copy a single (or 3) specific tar files e.g.
`gdas.t00z.satbias.tar.gz`, `gdas.t00z.satbias_cov.tar.gz`, and `gdas.t00z.tlapse.tar.gz` and let the jobs that need the specific ones untar these files.
It will help with keeping `COM` relatively clean of many files and eliminate the `glob` issue.
Thoughts?

Reply:
I agree with @CoryMartin-NOAA , @WalterKolczynski-NOAA , and you (@aerorahul ) that we can replace `glob` with a JEDI radiance bias correction tarball.  This removes `glob`, reduces the number of files in the `COM` directory, and simplifies file operations (e.g., a single `cp` gets all radiance files from _A_ to _B_).

This said, my preference is to make the change to a JEDI radiance tarball in a separate set of GDASApp and g-w issues and PRs.  g-w issue #2862 has been created with this approach in mind.

May this PR (#2833) move forward without including the JEDI radiance tarball changes?  Is this acceptable?

Reply:
Yes. That is perfectly fine. 
Thanks.